packages/kit/src/runtime/client/client.js

```diff
@@ -2027,7 +2027,7 @@ function _start_router() {
 	}
 
 	/** @param {MouseEvent} event */
-	container.addEventListener('click', (event) => {
+	container.addEventListener('click', async (event) => {
```

Comment:
I think this may cause errors to be ignored because `addEventListener` isn't async and doesn't accept a promise and so there's no rejection handler that gets applied in the error case